include/mbedtls/mbedtls_config.h

```diff
@@ -3680,8 +3680,56 @@
 
 /* Platform options */
 //#define MBEDTLS_PLATFORM_STD_MEM_HDR   <stdlib.h> /**< Header to include if MBEDTLS_PLATFORM_NO_STD_FUNCTIONS is defined. Don't define if no header is needed. */
-//#define MBEDTLS_PLATFORM_STD_CALLOC        calloc /**< Default allocator to use, can be undefined */
-//#define MBEDTLS_PLATFORM_STD_FREE            free /**< Default free to use, can be undefined */
+
+/* An overview of how the value of mbedtls_calloc is determined:
```

Comment:
This should be a Doxygen comment. Doxygen won't show the indentation, so the formatting should use bullet points. Attached to `MBEDTLS_PLATFORM_MEMORY`, I guess, since nothing interesting happens without it?